RockWeb/Blocks/Reporting/DynamicChart.ascx.cs

```diff
@@ -69,7 +69,7 @@ PageView pv
 </pre>
 </code>",
               CodeEditorMode.Sql )]
-
+    [TextField( "SQL Params", "Parameters to pass to query", false, "" )]
```

Comment:
For consistency with the DynamicData block change:
* SQL Params to Query Params
* Change description text to match the corresponding help text found in the tbParams (RockTextBox) of the DynamicData.ascx:  "The parameters that the stored procedure expects in the format of 'param1=value;param2=value'. Any parameter with the same name as a page parameter (i.e. querystring, form, or page route) will have it's value replaced with the page's current value. A parameter with the name of 'CurrentPersonId' will have it's value replaced with the currently logged in person's id."